mp4/samplegroupentries.go

```diff
@@ -149,3 +152,147 @@ func (s *UnknownSampleGroupEntry) Info(w io.Writer, specificBoxLevels, indent, i
 	}
 	return bd.err
 }
+
+// RollSampleGroupEntry - Grouping Type "roll"
+type RollSampleGroupEntry struct {
```

Comment:
May be good to mention that this is for "gradual decoding refresh" and have a reference to ISO/IEC 14496-12 Ed. 6 2020 Section 10.1